docs/iris/src/whatsnew/3.0.rst

```diff
@@ -1,12 +1,44 @@
 .. include:: ../common_links.inc
 
-v3.0 (01 Oct 2020)
+v3.0 (02 Oct 2020)
 ******************
 
 This document explains the changes made to Iris for this release
 (:doc:`View all changes <index>`.)
 
 
+.. dropdown:: :opticon:`report` Release Highlights
+   :container: + shadow
+   :title: text-primary text-center font-weight-bold
+   :body: bg-light
+   :animate: fade-in
+
+   The highlights for this major release of Iris include:
+
+   * We've finally dropped support for ``Python 2``, so welcome to ``Iris 3``
+     and ``Python 3``!
+   * Our :ref:`documentation <whatsnew 3.0 docs>` has been refreshed,
+     restructured, revitalised and rehosted on `readthedocs`_,
+   * It's now easier than ever to :ref:`install Iris <installing_iris>`
+     as a user or a developer, and the developers guide walks you though
+     how you can :ref:`get involved <development_where_to_start>` and
+     contribute to Iris,
+   * We've extended our coverage of the `CF Conventions and Metadata`_ by
+     introducting support for `CF Ancillary Data`_ and `Quality Flags`_,
+   * Lazy regridding is now available for several regridding schemes,
+   * We've introduced a common metadata API to simplify and unify the
+     management of metadata across Iris,
+   * Cube maths has been significantly improved with regards to extended
```

Comment:
maybe "cube arithmetic" rather than "cube maths" as maths could include all sorts of calculations (stats, regridding,...)